src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStaticImportCheck.java

```diff
@@ -70,6 +77,13 @@
  *   &lt;property name="excludes" value="java.lang.System.out,java.lang.Math.*"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * import static java.lang.Math.*;   // OK
+ * import static java.lang.System.*; // violation
```

Comment:
Description says
```
 To configure the check so that the {@code java.lang.System.out} member and all
 * members from {@code java.lang.Math} are allowed
```
Please change this line to java.lang.System.out and one more line with violation